epub33/a11y/index.html

```diff
@@ -915,83 +925,96 @@ <h6>Skippability</h6>
 										colours or placement so readers can filter this information visually out while
```

Comment:
> such as sidebars and footnotes

Suggest change to "print page numbers and footnotes".  Trying to put myself in the reader's position, I would not want to skip a sidebar but I would probably want to escape it.

Reply:
I'll get rid of that pesky "colours" while I'm at it. (Try as I might to misspell words like Americans... 😉 )